src/cmd/install.ts

```diff
@@ -12,7 +12,7 @@ export default async function installCmd (input: string[], opts: PnpmOptions) {
   input = input.filter(Boolean)
 
   const prefix = opts.prefix || process.cwd()
-  opts.hooks = requireHooks(prefix)
+  opts.hooks = requireHooks(prefix, opts)
```

Comment:
I know it currently means more code but it will better to add if statements outside, instead of passing in options to `requireHook()`. 

```js
if (!opts.ignorePnpmfile) {
  opts.hooks = requireHooks(prefix)
}
```